### PR TITLE
add package.json so it can be published to npm

### LINKED ISSUE #366: NPM package
Hello

Do you plan to publish FiraCode on NPM?

Thank you.

### PR DESCRIPTION
**Pull request by MasseGuillaume**
fix #366

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "firacode",
```
Comment (MasseGuillaume) [inline package.json]: needs to be lowercase

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "version": "1.205.0",
```
Comment (MasseGuillaume) [inline package.json]: needs to use semvar

**Comment by tonsky** [comment]
What will go into the package?
Where does 1.205.0 comes from?

**Comment by MasseGuillaume** [comment]
> What will go into the package?

All the repo! Maybe we should limit to distr folder.

> Where does 1.205.0 comes from?

The last version was 1.204. NPM only accept semver.org so it has to be 1.204.0 or something. Then I just bump one version up since 1.205 is already published.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tonsky/FiraCode.git"
+  },
+  "author": {
+    "name": "Nikita Prokopov",
+    "email": "prokopov@gmail.com"
+  },
+  "license": "SIL OPEN FONT LICENSE Version 1.1",
```
Comment (wopian) [inline package.json]: Should use `OFL-1.1` for the identifier

**Comment by MasseGuillaume** [comment]
@wopian Anything required on my side?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "font",
+    "Mozilla Fira Type Family",
+    "Fira",
+    "FiraMono"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tonsky/FiraCode.git"
+  },
+  "author": {
```
Comment (wopian) [inline package.json]: This is probably better to use the short hand version:

```json
"author": "Nikita Prokopov <prokopov@gmail.com>"
```

File: package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "Fira Code: monospaced font with programming ligatures",
+  "keywords": [
+    "font",
+    "Mozilla Fira Type Family",
+    "Fira",
+    "FiraMono"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tonsky/FiraCode.git"
```
Comment (wopian) [inline package.json]: Remove `git+` from the URL

**Comment by MasseGuillaume** [comment]
Done ;-)

**Comment by MasseGuillaume** [comment]
Reminder: All issues addressed here. Ready for merging :)

**Comment by ShadowRZ** [comment]
Should we also make it a Bower package?

**Comment by MasseGuillaume** [comment]
Bower is less and less used. NPM should be enough.

**Comment by wopian** [comment]
Bower themselves suggest you to use Yarn, which uses npm, on their [landing page](https://bower.io/)

**Comment by ShadowRZ** [comment]
Oh.
But does webpack support fonts?

**Comment by wopian** [comment]
Webpack works fine with fonts with either extract-text or [font-loader](https://github.com/izaakschroeder/font-loader). 😃

**Comment by tfrommen** [comment]
Oh yes, please, can we get this merged? :)

**Comment by ShadowRZ** [comment]
Oh, @wopian , I want know how to use this font in Visual Studio 2017 and 2015.

**Comment by wopian** [comment]
@ShadowRZ for 2017, `Tools > Options > Environment > Fonts and Colours`

**Comment by ShadowRZ** [comment]
I did.
But how do I make font ligatures?

于 2017年4月4日 GMT+0800PM1时48分31秒, James Harris <notifications@github.com> 写到:
>@ShadowRZ for 2017, `Tools > Options > Environment > Fonts and Colours`
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/tonsky/FiraCode/pull/374#issuecomment-291399812

-- 
使用 K-9 Mail 发送自我的手机的说～
来自NicoNicoNi的HID_System的说～

**Comment by tfrommen** [comment]
Was that a mistake, @MasseGuillaume? 🤔

**Comment by MasseGuillaume** [comment]
@tfrommen I don't want to loose my time here.

**Comment by tfrommen** [comment]
Yeah, well, I feel with you, @MasseGuillaume. But how does leaving this open waste any of your time?

I mean, now it'd mean **two** steps (instead of just one) to get this merged - if at all, I know.

**Comment by tonsky** [comment]
I’ll get it merged, eventually, let’s leave it open for now

**Comment by tfrommen** [comment]
What's in the way for this to get merged, now?

Genuine question. If something is unclear, @tonsky, I'm sure people will gladly try to get rid of that (myself included).

**Comment by tonsky** [comment]
Just priorities :)

**Comment by rriemann** [comment]
Let me first thank you for this nice project. I just discovered Fira Code. @tonsky , can you press the merge button?

**Comment by tonsky** [comment]
Sorry everyone it took so long. Hope it helps!

**Pull request merged**